src/react/workflow/ExpirationForm.tsx

```diff
@@ -56,11 +56,21 @@ const PluralizeDays = ({ number }: { number: number | string }) => {
   );
 };
 
+const hasUniqueKeys = (value: any, helper: any, options = { keyName: 'key'}) => {
```

Comment:
```suggestion
const hasUniqueKeys = (value: {key: string, value: string}[], helper: CHANGE_ME_REACT_HOOK_FORM_HELPER_TYPE, options: {keyName: string} = { keyName: 'key'}) => {
```

Additionally `{key: string, value: string}` could be extracted to a Tag type if not already existing